tests/tenant_test.go

```diff
@@ -570,3 +570,246 @@ func TestTenantDatabaseNameTemplate(t *testing.T) {
+		}
+	}
+	for _, prodInstance := range prodInstances {
+		for _, database := range databases {
+			if database.Instance.ID == prodInstance.ID {
+				prodDatabases = append(prodDatabases, database)
+				break
+			}
+		}
+	}
+	require.Equal(t, len(stagingDatabases), stagingTenantNumber)
+	require.Equal(t, len(prodDatabases), prodTenantNumber)
+
+	// Simulate Git commits.
+	gitFile := "bbtest/testTenantVCSSchemaUpdate__ver1__migrate__create_a_test_table.sql"
+	pushEvent := &gitlab.WebhookPushEvent{
+		ObjectKind: gitlab.WebhookPush,
+		Ref:        "refs/heads/feature/foo",
+		Project: gitlab.WebhookProject{
+			ID: gitlabProjectID,
+		},
+		CommitList: []gitlab.WebhookCommit{
+			{
+				Timestamp: "2021-01-13T13:14:00Z",
+				AddedList: []string{
+					gitFile,
+				},
+			},
+		},
+	}
+	err = ctl.gitlab.AddFiles(gitlabProjectIDStr, map[string]string{gitFile: migrationStatement})
+	require.NoError(t, err)
+	err = ctl.gitlab.SendCommits(gitlabProjectIDStr, pushEvent)
+	require.NoError(t, err)
+
+	// Get schema update issue.
+	openStatus := []api.IssueStatus{api.IssueOpen}
+	issues, err := ctl.getIssues(api.IssueFind{ProjectID: &project.ID, StatusList: &openStatus})
+	require.NoError(t, err)
+	require.Equal(t, len(issues), 1)
+	issue := issues[0]
+	status, err := ctl.waitIssuePipeline(issue.ID)
+	require.NoError(t, err)
+	require.Equal(t, status, api.TaskDone)
+
+	// Query schema.
+	for i, stagingInstance := range stagingInstances {
+		tenant := fmt.Sprintf("tenant%d", i)
+		databaseName := baseDatabaseName + "_" + tenant
+		result, err := ctl.query(stagingInstance, databaseName, bookTableQuery)
+		require.NoError(t, err)
+		require.Equal(t, bookSchemaSQLResult, result)
+	}
+	for i, prodInstance := range prodInstances {
+		tenant := fmt.Sprintf("tenant%d", i)
+		databaseName := baseDatabaseName + "_" + tenant
+		result, err := ctl.query(prodInstance, databaseName, bookTableQuery)
+		require.NoError(t, err)
+		require.Equal(t, bookSchemaSQLResult, result)
+	}
+
+	// Query migration history
+	hm1 := map[string]bool{}
+	hm2 := map[string]bool{}
+	for i, instance := range stagingInstances {
+		tenant := fmt.Sprintf("tenant%d", i)
+		databaseName := baseDatabaseName + "_" + tenant
+		histories, err := ctl.getInstanceMigrationHistory(db.MigrationHistoryFind{ID: &instance.ID, Database: &databaseName})
+		require.NoError(t, err)
+		require.Equal(t, len(histories), 2)
+		require.Equal(t, histories[0].Version, "ver1")
+		require.NotEqual(t, histories[1].Version, "")
+		hm1[histories[0].Version] = true
+	}
+	for i, instance := range prodInstances {
+		tenant := fmt.Sprintf("tenant%d", i)
+		databaseName := baseDatabaseName + "_" + tenant
+		histories, err := ctl.getInstanceMigrationHistory(db.MigrationHistoryFind{ID: &instance.ID, Database: &databaseName})
+		require.NoError(t, err)
+		require.Equal(t, len(histories), 2)
+		require.Equal(t, histories[0].Version, "ver1")
+		require.NotEqual(t, histories[1].Version, "")
+		hm2[histories[0].Version] = true
+	}
+
```

Comment:
nit: we can check the latestSchemaFile in the test.

Reply:
done